File: baekjoon/silver/1193/1193.go
package main

import "fmt"

func main() {
	var x int

	fmt.Scan(&x)

	i := 0
	for {
		if i*(i+1)/2 >= x && i*(i-1)/2 < x {
			break
		}
		i++
	}
	step := x - i*(i-1)/2
	if i%2 == 0 {
		fmt.Printf("%d/%d\n", step, i-step+1)
	} else {
		fmt.Printf("%d/%d\n", i-step+1, step)
	}
}

File: baekjoon/silver/1193/1193.py
# input
# 14
# output
# 2/4

x = int(input())
i = 0
while True:
    if i*(i+1)//2 >= x and i*(i-1)//2 < x:
        break
    i += 1
step = x-i*(i-1)//2
if i % 2 == 0:
    print(str(step)+"/"+str(i-step+1))
else:
    print(str(i-step+1)+"/"+str(step))
